src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  margin: 0; /* Скидання стандартних відступів */
+  padding: 0; /* Скидання стандартних відступів */
+  box-sizing: border-box; /* Включає padding та border в ширину та висоту */
+}
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif; /* Використання шрифту Roboto */
+  background-color: #858585; /* Колір фону */
+}
+
+.header {
+  display: flex;
+  align-items: center; /* Вертикальне центровання */
+  justify-content: space-between; /* Розташування логотипу та навігації */
+  background-color: #fff; /* Колір фону заголовка */
+  height: 60px; /* Висота заголовка */
+  padding: 0 50px; /* Відступи з боків для заголовка */
+}
+
+.logo img {
+  height: 40px; /* Висота логотипу */
+  margin-top: 10px;
+}
+
+nav {
+  display: flex; /* Гнучке розташування для навігації */
+}
+
+nav ul {
+  display: flex; /* Горизонтальне розташування списку */
+}
+
+.nav__item {
+  list-style: none; /* Прибирає маркери списку */
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 30px;
+}
+
+.nav__link {
+  word-spacing: nowrap; /* Видалення переносу слів для лаптопів і тдю */
+  text-transform: uppercase; /* Великі літери для тексту */
+  color: black; /* Колір тексту */
+  text-decoration: none; /* Без підкреслення */
+  display: flex; /* Використовує flexbox для центровання тексту */
+  align-items: center; /* Центрує текст вертикально */
+  position: relative;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link1 {
+  word-spacing: nowrap; /* Видалення переносу слів для лаптопів і тдю */
+  text-transform: uppercase; /* Великі літери для тексту */
+  color: #00acdc; /* Колір тексту */
+  text-decoration: none; /* Без підкреслення */
+  display: flex; /* Використовує flexbox для центровання тексту */
+  align-items: center; /* Центрує текст вертикально */
+  height: 14px; /* Висота лінків */
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+}
```

Comment:
The '.nav__link1' class appears to be redundant as it shares many styles with '.nav__link'. Consider merging these classes to avoid duplication and improve maintainability.